.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+## Checklist
+
+<!---
+	Put an `x` in all the boxes that apply
+-->
+
+- [ ] I read and understood the **[Contributing Guidelines](https://github.com/JustArchiNET/ArchiSteamFarm/blob/main/.github/CONTRIBUTING.md)**.
+- [ ] This is not a **[duplicate](https://github.com/JustArchiNET/ArchiSteamFarm/pulls)** of an existing merge request.
+- [ ] I believe this falls into the scope of the project and should be part of the built-in functionality.
+- [ ] My code follows the **[code style](https://github.com/JustArchiNET/ArchiSteamFarm/blob/main/.github/CONTRIBUTING.md#code-style)** of this project.
+- [ ] I have added tests to cover my changes, wherever they are necessary.
+- [ ] All new and existing tests pass.
+
+## Changes
+
+### New functionality
+
+<!---
```

Comment:
I dislike boxes like that, because people might put what they want in the `Please describe here, what new functionality was added.`, and the whole block will be commented out.

Instead, I'd want to see entries like `<!-- Please describe below, what new functionality was added. -->`. This way decreases the probability of user putting his changes in the comment, and regardless whether he removes it or not, it won't be displayed.

Also notice `<!--` instead of `<!---`.

Refer to **[removed enhancement](https://github.com/JustArchiNET/ArchiSteamFarm/commit/39d5f9c348d79b9d3241c6644c0f7a9c80dc7e42#diff-e8f8e6b9a18f99556e2bb6ed0b81cceba9ac6524acc27bc6044bd19038633866)** feature request for further details.

Reply:
Also, add `Additional info` section.